functions/requests/routes/users.js

```diff
@@ -1,293 +1,35 @@
 const express = require('express');
 const admin = require('firebase-admin');
-const cors = require('cors')({ origin: true });
-const Busboy = require('busboy');
-const path = require('path');
-const os = require('os');
-const fs = require('fs');
+const uuid = require('uuid/v4');
```

Comment:
We're getting some console warnings because this way of importing uuid is deprecated. [uuid](https://www.npmjs.com/package/uuid)